gdal/ogr/ogrsf_frmts/xlsx/ogrxlsxdatasource.cpp

```diff
@@ -639,25 +647,13 @@ void OGRXLSXDataSource::startElementTable(const char *pszNameIn,
             return;
         }
         nNewCurLine --;
-        const int nFields = std::max(
```

Comment:
Why has been this logic removed ? This helps fixing various timeouts / out of memory situation on fuzzed files